tensorflow_datasets/image/imagenette.py

```diff
@@ -0,0 +1,139 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Imagenette: a subset of 10 easily classified classes from Imagenet (tench,
+English springer, cassette player, chain saw, church, French horn, garbage
+truck, gas pump, golf ball, parachute)"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import io
+import itertools
+import os
+
+import numpy as np
+import tensorflow as tf
+
+import tensorflow_datasets.public_api as tfds
+
+# TODO(imagenette): BibTeX citation
+_CITATION = """
```

Comment:
I still have to add citation as I couldn't find one on the website.

Reply:
You can contact the author of the dataset and ask if they have one.

Reply:
https://github.com/fastai/imagenette/issues/16